src/stylesheets/mixins.scss

```diff
@@ -0,0 +1,11 @@
+/// Slightly lighten a color
+/// @access public
+/// @param {Color} $color - color to tint
+/// @param {Number} $percentage - percentage of `$color` in returned color
+/// @return {Color}
+
+$white: #FFF !default;
```

Comment:
You can define this in `colors.scss` and rearrange the order of `mixins` and `colors` in the main scss file.

Reply:
Turns out I can't; `colors` uses the `tint` method (which uses `$white`) from `mixins`...so if `mixins` needs to import `$white` from `colors`, we've got an endless loop...